models/productDao.js

```diff
@@ -0,0 +1,39 @@
+const appDataSource = require("./appDataSource");
+
+const getProducts = async (req) => {
+  return await appDataSource.query(
+    `SELECT
+      name,
+      price,
+      thumbnail_image
+      FROM products`
+  );
+};
+
+const getProductInfo = async (productId) => {
+  return await appDataSource.query(
+    `SELECT
+      name,
+      price,
+      thumbnail_image,
+      product_images.image_url
+      FROM products
+      join product_images on products.id = product_images.product_id
+      where product_id = ?`,
+    [productId]
+  );
+};
+
+const getProductTag = async (tagId) => {
```

Comment:
해당 method는 query문을 보면 productTag를 get하는 것이 아닌, products에 대한 정보를 part_tag_id를 통해 가져오는 것 같습니다. 이에 맞게 수정해주세요~
```suggestion
const getProductsByPartTagId = async (tagId) => {
```

Reply:
네 알아볼수있게 변수명 수정했습니다 !